### ISSUE TITLE
Pull Changes - conflict resolution shows nothing in the comparison screens

### ISSUE THREAD

**Issue by mhmacleod** [a85/POSTMan-Chrome-Extension#9786]
The sections that are supposed to show the conflicting code to resolve are all blank. selecting either does not show a different result on the final code
![image](https://user-images.githubusercontent.com/16532819/113109604-b6f0f280-9238-11eb-87ac-4bab85c67381.png)

**Comment by DannyDainton** [comment]
Hey @mhmacleod 

Could you provide some more context and some steps to reproduce the problem, please?

The Postman version will also help here too. 🏆

**Comment by mhmacleod** [comment]
Hi - Postman desktop was latest at time of issue (~8.1.0), but we also used the website for good measure with the same results.
Issue happened by forking branch B from A, then trying to merge A down to B before trying to merge back the final result to A.
The first step (get latest) hit the above issue so we couldn't proceed any further.

However when I checked both today, the function now appears to be working well now, without having changed anything (I think). Perhaps something got fixed on the Postman servers.
